dataStandard_versioned_docs/version-5/whats-new/api_changes_5.md

```diff
@@ -0,0 +1,304 @@
+
+* **POST** /ed-fi/studentSectionAssociations
+
+   **Request**
+   * Added property: dualCreditEducationOrganizationReference (object)
+   * Added property: dualCreditIndicator (boolean)
+   * Added property: dualCreditInstitutionDescriptor (string)
+   * Added property: dualCreditTypeDescriptor (string)
+   * Added property: dualHighSchoolCreditIndicator (boolean)
+
+* **GET** /ed-fi/studentSectionAssociations/{id}
+
+   **Response**
+   * Added property: dualCreditEducationOrganizationReference (object)
+   * Added property: dualCreditIndicator (boolean)
+   * Added property: dualCreditInstitutionDescriptor (string)
+   * Added property: dualCreditTypeDescriptor (string)
+   * Added property: dualHighSchoolCreditIndicator (boolean)
+
+* **PUT** /ed-fi/studentSectionAssociations/{id}
+
+   **Request**
+   * Added property: dualCreditEducationOrganizationReference (object)
+   * Added property: dualCreditIndicator (boolean)
+   * Added property: dualCreditInstitutionDescriptor (string)
+   * Added property: dualCreditTypeDescriptor (string)
+   * Added property: dualHighSchoolCreditIndicator (boolean)
+
+### [Alternative and Supplemental Services Domain](https://docs.ed-fi.org/reference/data-exchange/data-standard/model-reference/alternative-and-supplemental-services-domain/best-practices)  
+
+#### Add Shortened School Days for IEP student to studentSpecialEducationProgramAssociations
+
+* **GET** /ed-fi/studentSpecialEducationProgramAssociations
+
+   **Parameters**
+   * **Add** reductionInHoursPerWeekComparedToPeers //Records the number of hours reduced for the shortened school day for the IEP student as compared to peers in regular education. //Indicator that the student's IEP requires a shortened school day.
+   * **Add** shortenedSchoolDayIndicator //The reason the student left the program within a school or district.
+   * **Add** reasonExitedDescriptor 
+   * **Add** endDate //Indicates whether the student received services during the summer session or between sessions.
+   * **Add** servedOutsideOfRegularSession //Indicates whether the student received services during the summer session or between sessions.
+
+   **Response** 
+   * Added property: [n].reductionInHoursPerWeekComparedToPeers (number)
+   * Added property: [n].shortenedSchoolDayIndicator (boolean)
+   * Changed property: [n].beginDate (string)
+   * Changed property: [n].endDate (string)
+   * Changed property: [n].iepBeginDate (string)
+   * Changed property: [n].iepEndDate (string)
+   * Changed property: [n].programParticipationStatuses (array)
+   * Changed property: [n].specialEducationProgramServices (array)
+   * Changed property: [n].programParticipationStatuses[n].statusBeginDate (string)
+   * Changed property: [n].programParticipationStatuses[n].statusEndDate (string)
+   * Changed property: [n].specialEducationProgramServices[n].serviceBeginDate (string)
+   * Changed property: [n].specialEducationProgramServices[n].serviceEndDate (string)
```

Comment:
What changed about these properties? For the dates, was it only the note we have at the bottom of the page? If so, I wonder if we can remove the lines saying that the dates changed?

And for others - programParticipationStatuses and specialEducaitonProgramServices - can we describe exactly what changed?